### ISSUE TITLE
Feature recommendation: syntax highlighting

### ISSUE THREAD

**Issue by axelbsa** [dr-soft/dred#11]
Hey.
First, like your editor very much.

Would it be possible to implement some rudimentary syntax highlighting?

**Comment by mackron** [comment]
Yes, a simple syntax highlighting system is on my TODO list. I'll leave this issue open so people who are interested in this can track it's progress.
